scripts/zones/Alzadaal_Undersea_Ruins/npcs/_20m.lua

```diff
@@ -106,12 +106,12 @@ function onEventUpdate(player,csid,option,target)
             for i,v in ipairs(party) do
                 if (v:getID() ~= player:getID()) then
                     if (v:hasKeyItem(NYZUL_ISLE_ROUTE) == false and v:hasCompletedMission(TOAU, PATH_OF_DARKNESS) == false) then
-                        player:messageText(target,MEMBER_NO_REQS, false);
-                        player:instanceEntry(target,1);
+                        v:messageText(target,MEMBER_NO_REQS, false);
```

Comment:
these changes aren't right - it should inform the player trying to register the instance when a party member doesn't meet the requirements, not the party member

Reply:
(you left all the other ones correct though, just this one is wrong)